### PR TITLE
feat: Add copy token address button to currency input panel

### PR DESCRIPTION
**Pull request by memoyil**
To review 

1. Go to swap
2. Select token from list
3. See copy address button

### CODE REVIEW

**Comment by chef-jojo** [comment]
Another good feature, thank you! I was thinking about the same thing. 
I will pass it to design team to make it more polish

**Comment by chefilip** [comment]
I think we could add `CheckmarkIcon` to make it more illustrativ
CleanShot 2022-02-25 at 09 45 16
e

**Comment by chefilip** [comment]
Screen Shot 2022-02-25 at 09 48 23

Could we add token address for BNB ?

**Comment by chef-jojo** [comment]
> Screen Shot 2022-02-25 at 09 48 23
> 
> Could we add token address for BNB ?

Native currency should be configured by wallet itself

File: src/components/CurrencyInputPanel/index.tsx [hunk 1]
```diff
@@ -90,57 +91,68 @@ export default function CurrencyInputPanel({
...
+                      )}`
+                    : currency?.symbol) || t('Select a currency')}
+                </Text>
+              )}
+              {!disableCurrencySelect && <ChevronDownIcon />}
+            </Flex>
+          </CurrencySelectButton>
+          {token && tokenAddress ? (
+            <Flex style={{ gap: '4px' }} alignItems="center">
+              <CopyButton
```
Comment (chef-jojo) [inline src/components/CurrencyInputPanel/index.tsx:131]: Could you make it button color to `textSubtle`?

Comment (memoyil) [reply src/components/CurrencyInputPanel/index.tsx:134]: Done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**